src/components/value-or-loading/ValueOrLoading.jsx

```diff
@@ -11,7 +11,7 @@ function LoadingOrValue({ value }) {
 }
 
 LoadingOrValue.propTypes = {
-  value: PropType.element,
+  value: PropType.oneOfType([PropType.element, PropType.number]),
```

Comment:
I think having it be a node means it can be anything that can be rendered, so it will match, numbers, strings and other react elements. 
```suggestion
  value: PropType.node,
```

Reply:
Changed.